src/references/auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Authentication
+---
+
+## Authentication Sequence
+
+1. The User opens up a browser to check his cozy, using his *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+2. he's redirected to his *cozy user domain* (`user.mycozy.cloud`)
+3. The browser request the *cozy user domain*, which send back the authentication form. The user fills his password, sending it back to the  *cozy user domain*
+4. The server *cozy user domain* set an auth cookie for *cozy user domain* ONLY
+5. A redirect status response code (302) for the requested app domain (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...) is sent to the browser
+5. The browser request the *cozy app domain* with a [jwt code](https://jwt.io/)
```

Comment:
Well, it's a detail but the code used to transfer the session from the cozy user domain to the cozy app domain is not a JWT (just a random string)

Reply:
The JWT token is not used for transfering the cookie from one domain to another: it's the random code that is used for that. And when the cookie is set on the app domain, the app can fetch its index.html page with the JWT token inside it, and extracts it in JS.